### ISSUE TITLE
Questions about Fonts100's test set

### ISSUE THREAD

**Issue by Jeozhao** [liweileev/FET-GAN#2]
Hi @liweileev ，thank you for your paper code.  I get some questions about Fonts100's test set. Is the style of the source of this test set randomly selected?
Thank you!

**Comment by liweileev** [comment]
At the test stage, the style images are randomly selected images from the trained styles. As described in the paper, we use 80 fonts for training and each of them contains 837 images. For the test (not finetune), style images are selected randomly from the 837 images in the same one style.

**Comment by liweileev** [comment]
In fact, the style image only provides the style information. So it doesn't matter which character it is. Theoretically, any character under the same style will lead to the same transfer result.
